### ISSUE TITLE
Add locale-dependent presentation for memento count

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#779]
<img width="1484" alt="Screen Shot 2022-09-29 at 12 32 41 PM" src="https://user-images.githubusercontent.com/2514780/193088177-108aa643-6dd2-43f7-a540-bf181d323b9f.png">

For en-us, the `50000` here should read `50,000` but that presentation will depend on the locale of the client.

This was generated using `python3 cdxjGenerator/cdxjGenerator.py 50000 > ./test.cdxj` then `ipwb replay test.cdxj` with current main branch head 2094cd2a32f1e179b19b0d51eb8b02026307d303.

**Comment by machawk1** [comment]
```py
import locale
from decimal import Decimal

local.setlocale(locale.LC_NUMERIC, 'en-US') 
# ^ read server locale rather than hard-coding

print(f'{Decimal("50000"):n}')
```

This makes me think that it should be formatted on the client-side. The `count` element is destined for the webadmin interface but this might be applicable elsewhere, e.g., the replay banner for the client.
